### DISCUSSION TITLE
Occasional black screen from Plasma/Filer startup order

### DISCUSSION THREAD

**Discussion by grahamperrin** [ravynsoft/ravynos#140] [Q&A] [answered]
https://github.com/mszoek/airyx/projects/2#card-68221816 – please, is that still an issue? 

I see https://github.com/mszoek/airyx/issues/102 closed.

**Comment by markshank** [comment]
Yes, it is still an issue.  It remains on the project To Do list.

You can work around it with an Alt+Tab from the keyboard IIRC.

**Reply by mszoek** [reply]
Yes, unfortunately it is still an issue. Alt+Tab or kill and restart Filer with `killall Filer; open -a Filer` should resolve it.

**Reply by grahamperrin** [reply]
Still an issue?

**Reply by mszoek** [reply]
No. We moved away from KDE/X11 and Filer, so this issue is no longer relevant.

**Comment by grahamperrin** [comment, marked as answer]
Answered: https://github.com/ravynsoft/ravynos/discussions/140#discussioncomment-3887938 thanks @mszoek
